book/src/generalized-ownership.md

```diff
@@ -0,0 +1,279 @@
+# Generalized Ownership
+
+This design describes generalized ownership of countable items on the Solana
```

Comment:
I think this proposal should start off with a problem description. It's not clear why the reader should read anything beyond this first line. The second line suggests the problem being solved is code duplication, which isn't terribly interesting on its own. But if the problem being solved is the long-standing problem that a batch of credits to the same user-defined token account (like ERC20) is slower than credits to lamports, then the value-add is very clear. Also, if that's the context, maybe generalizing the ownership model is a sledgehammer of a solution and perhaps there's better ROI in a more conservative proposal.

Reply:
@garious, I am not sure how to implement an exchange program that can support user defined tokens without something like this.